deployments/common/zeppelin/sql/auth.sql

```diff
@@ -0,0 +1,27 @@
+USE zeppelin;
+CREATE TABLE users (username TEXT, password TEXT, password_salt TEXT);
```

Comment:
The `users` table has a column for `salt` but we don't use it in any of the inserts. Fine for now but we should probably look at using it to add another level of security in the future.

Reply:
I haven't addressed this in this fork / PR. Perhaps an improvement we can add later?

Reply:
Yep, fine for now.